### PR TITLE
added hypothesis-csv to docs

### PR DESCRIPTION
**Pull request by chobeat**

### CODE REVIEW

File: hypothesis-python/docs/strategies.rst [hunk 1]
```diff
@@ -25,7 +25,8 @@ Some packages provide strategies directly:
   `D-Bus signatures <https://dbus.freedesktop.org>`_.
 * :pypi:`hypothesis_sqlalchemy` - strategies to generate :pypi:`SQLAlchemy` objects.
 * :pypi:`hypothesis-ros` - strategies to generate messages and parameters for the `Robot Operating System <http://www.ros.org/>`_.
-
+* :pypi:`hypothesis-csv` - strategy to generate CSV files.
+*
```
Comment (alexwlchan) [inline hypothesis-python/docs/strategies.rst]: Docs build is complaining about this bullet; it should be a blank line.

Comment (chobeat) [reply hypothesis-python/docs/strategies.rst]: can you please tell me how to reproduce the build locally to validate? I couldn't find it in the docs

Comment (chobeat) [reply hypothesis-python/docs/strategies.rst]: or should I just go through the ci?

Comment (chobeat) [reply hypothesis-python/docs/strategies.rst]: anyway I pushed the fix

**Pull request merged**